packages/lab/src/layout/utils/useIsViewportLargerThanBreakpoint.ts

```diff
@@ -4,15 +4,17 @@ import {
   useOrderedBreakpoints,
 } from "@jpmorganchase/uitk-core";
 
-export const useIsStacked = (stackedAtBreakpoint: keyof Breakpoints) => {
+export const useIsViewportLargerThanBreakpoint = (
```

Comment:
should this not be in the general utils? i guess if there is nothing else that will use it it makes sense here, not sure if there would be other use cases

Reply:
Makes sense, I think this could be used in other components like `ButtonBar` so I've moved it to the general utils folder